packages/persist/lib/routes/getHealth.ts

```diff
@@ -0,0 +1,18 @@
+import type { Endpoint } from '@nangohq/types';
+import type { RouteHandler } from '@nangohq/utils';
+
+type Health = Endpoint<{
+    Method: typeof method;
+    Path: typeof path;
+    Success: { status: 'ok' };
+}>;
+
+const path = '/health';
+const method = 'GET';
+
+export const routeHandler: RouteHandler<Health> = {
+    path,
+    method,
+    validate: (_req, _res, next) => next(), // No extra validation needed
```

Comment:
imo the opposite of validation is not 'no validation' but ensure it's empty

Reply:
the emphasis is on the `extra` but I understand it might be understood differently. will remove the comment that doesn't bring much info anyway

Reply:
ah yes thanks